### PR TITLE
pad the output in `logConcurrentOutput` mode

### LINKED ISSUE #43: stdout isn't padded when logConcurrentOutput is true
```js
module.exports = function (grunt) {
    grunt.loadNpmTasks('grunt-concurrent');
    grunt.registerTask('log', 'Log to the console.', function (message) {
        console.log(message);
    });
    grunt.initConfig({
        concurrent: {
            logNo: {
                options: {
                    logConcurrentOutput: false
                },
                tasks: ['log:one', 'log:two']
            },
            logYes: {
                options: {
                    logConcurrentOutput: true
                },
                tasks: ['log:one', 'log:two']
            }
        }
    });
};
```

```shell
$: grunt concurrent:logNo
Running "concurrent:logNo" (concurrent) task

    Running "log:one" (log) task
    one

    Done, without errors.

    Running "log:two" (log) task
    two

    Done, without errors.

Done, without errors.
```
```shell
$: grunt concurrent:logYes
Running "concurrent:logYes" (concurrent) task
Running "log:two" (log) task
Running "log:one" (log) task
one
two


Done, without errors.
Done, without errors.





Done, without errors.
```

### LINKED ISSUE #55: Output isn't padded on Windows
Tested on Windows 7 x64, Node v0.10.31 and Windows 8.1 x64, Node v0.12.2 using example in #43. Here's the output I get:

```
>grunt concurrent:logNo
Running "concurrent:logNo" (concurrent) task

Running "log:two" (log) task
two

Done, without errors.

Running "log:one" (log) task
one

Done, without errors.

Done, without errors.
```

```
>grunt concurrent:logYes
Running "concurrent:logYes" (concurrent) task
Running "log:one" (log) task
one

Done, without errors.


Running "log:two" (log) task
two

Done, without errors.



Done, without errors.
```

### PR DESCRIPTION
**Pull request by sindresorhus**
fixes #43
fixes #55

screen shot 2015-06-15 at 11 51 47

// @kevva @arthurvr @stevemao @shinnn

### CODE REVIEW

**Comment by sindresorhus** [comment]
Gonna keep this open for a few days so people are able to try it out. Then I'm merging and doing a major release.

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   ],
   "dependencies": {
     "async": "^1.2.1",
-    "pad-stdio": "^1.0.0"
+    "pad-stdio": "^2.0.0"
```
Comment (stevemao) [inline package.json]: I think this needs to be ^2.0.1 as it is the minimum version for this to work properly.

Comment (sindresorhus) [inline package.json]: @stevemao This isn't `lpad` ;)

Comment (stevemao) [inline package.json]: Sorry :p

**Comment by kevva** [comment]
Does this look correct?

2015-06-16 17_48_38-online photo editor _ pixlr editor _ autodesk pixlr

**Comment by stevemao** [comment]
Is `autoprefixer` and `connect` under the concurrent task? @kevva 
Can you share you gruntfile

**Comment by kevva** [comment]
@stevemao, I just did a quick test with `generator-webapp` (https://github.com/yeoman/generator-webapp/blob/master/app/templates/Gruntfile.js#L417-L420). Only had `copy` so it seems right to me.

**Pull request closed without merging**